### PR TITLE
Add legend position + title and subtitle for choropleths

### PR DESCRIPTION
**Pull request by wmai**
## Summary

The purpose of this PR is to allow the positioning of the legend for the maps, as well as the addition of a title and a subtitle.

### Changelog

In the Map component: It's now possible to set the position of the legend and add a title or/and a subtitle.

## ⚠️ TODO BEFORE MERGE
- [x] Merge [the PR enabling value formatting](https://github.com/opendatasoft/ods-dataviz-sdk/pull/131) into this branch.

## Review checklist

- [x] Description is complete
- [x] 2 reviewers (1 if trivial)
- [ ] Tests coverage has improved
- [x] Code is ready for a release on NPM

### CODE REVIEW

File: packages/visualizations/src/components/utils/ColorsLegend.svelte [hunk 1]
```diff
@@ -45,84 +47,96 @@
...
-        </div>
-    {:else if colorScale.type === 'palette'}
-        <!-- Palette color boxes, row display, no labels only displaying palettes steps -->
-        <div class="legend-colors-container-palette" bind:clientWidth={legendWidth}>
-            <div class="legend-colors-row-color-box-palette">
-                {#each colorScale.colors as color}
-                    <div class="legend-colors-color-box-palette" style="--box-color: {color}" />
-                {/each}
-            </div>
+<div class={`legend-container legend-container--${position}`}>
```
Comment (etienneburdet) [inline packages/visualizations/src/components/utils/ColorsLegend.svelte:50]: The extra wrapper div is not necessary… but it doesn't hurt either 🤷 Probably make things easier to stabilize in the future and that's really just one div…

**Review by etienneburdet — APPROVED** [review]
LGTM! Title and subtitle seem easy enough to stlye if needed without adding any more selector/var… and it works 🏄

**Review by richterb — APPROVED** [review]

File: packages/visualizations/src/components/Map/types.ts [hunk 2]
```diff
@@ -28,6 +28,10 @@ export interface ChoroplethOptions {
     bbox?: BBox | undefined;
     /** Attribution to display on the map */
     attribution?: string;
+    /** Title of the map */
+    title?: string;
```
[the remark this answers is not in the document]
Comment (wmai) [reply packages/visualizations/src/components/Map/types.ts:32]: Looking at the code, I don't see any good reason to have this `display` prop. In the Svelte component, we could accomplish the same conditional display by checking if the string's empty or not.

[the remark this answers is not in the document]
Comment (wmai) [reply packages/visualizations/src/components/Map/types.ts:32]: In the Studio, we'll need to change a few functions, such as:

```TS
export function titleConfig(styles: Required<CartesianSeriesStyle | RadialSeriesStyle>) : TitleConfiguration {
    const { layout, title } = styles;
    return {
        display: !!LAYOUTS_BEHAVIOUR[layout][TITLE_TOP],
        text: title,
    };
}
```

into:

```TS
export function titleConfig(styles: Required<CartesianSeriesStyle | RadialSeriesStyle>) : string | undefined {
    const { layout, title } = styles;
    return !!LAYOUTS_BEHAVIOUR[layout][TITLE_TOP] ? title : undefined;
}
```

**Pull request merged**